packages/integrations/netlify/README.md

```diff
@@ -72,6 +72,24 @@ export default defineConfig({
 });
 ```
 
+### Per-page functions
+
+The Netlify adapter builds to a single function by default. Astro 2.9 added support for splitting your build into separate entry points per page. If you use this configuration the Vercel adapter will generate a separate function for each page. This can help reduce the size of each function so they are only bundling code used on that page.
```

Comment:
```suggestion
The Netlify adapter builds to a single function by default. Astro 2.9 added support for splitting your build into separate entry points per page. If you use this configuration, the Netlify adapter will generate a separate function for each page. This can help reduce the size of each function so they are only bundling code used on that page.
```
:smile: